examples/samplecontainer/contents/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+etcd-v3.2.9-linux-amd64/etcd &
+
+kubernetes/server/bin/kube-apiserver \
+  --authorization-mode=RBAC \
+  --insecure-bind-address 0.0.0.0 \
+  --etcd-servers http://127.0.0.1:2379 \
+  --service-cluster-ip-range=10.10.0.0/14 \
+  --enable-aggregator-routing \
+  --requestheader-allowed-names=aggregator \
+  --requestheader-extra-headers-prefix=X-Remote-Extra- \
+  --requestheader-group-headers=X-Remote-Group &
+
+./clusterregistry \
+  --secure-port 443 \
+  --etcd-servers http://127.0.0.1:2379 &
+
+sleep 5
```

Comment:
Is this enough? Is there any other way to make this resilient?

Reply:
Done.